### PR TITLE
Add jos-animation w/ npm auto-update

### PR DESCRIPTION
**Pull request by jesvijonathan**
Adds [JOS-Animation](https://jos-animation.vercel.app/).

### CODE REVIEW

File: packages/j/jos-animation.json [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    "type": "git",
+    "url": "git://github.com/jesvijonathan/JOS-Animation-Library.git"
+  },
+  "autoupdate": {
+    "source": "npm",
+    "target": "jos-animation",
+    "fileMap": [
+      {
+        "basePath": "dist",
+        "files": ["**/*"]
```
Comment (MattIPv4) [inline packages/j/jos-animation.json:38]:
```suggestion
        "files": [
          "*.@(css|js)"
        ]
```
Please avoid using open globstars, as they can lead to including files not intended for the CDN. I suspect you just need the CSS/JS files directly in the `dist` directory?

(Also, new line for each item in array, allows for cleaner diffs in the future)

File: packages/j/jos-animation.json [hunk 2]
```diff
@@ -0,0 +1,43 @@
+{
+  "name": "jos-animation",
+  "description": "An animation library package to instantly add beautiful/professional looking animation to your website",
+  "homepage": "jos-animation.vercel.app",
+  "filename": "dist/jos.js",
```
Comment (MattIPv4) [inline packages/j/jos-animation.json:5]:
```suggestion
  "filename": "jos.js",
```
As the `basePath` is set to `dist`, we'll pull the files out of that directory into the root.

**Review by MattIPv4 — CHANGES_REQUESTED** [review]

File: packages/j/jos-animation.json (hunk 1 above)
Comment (jesvijonathan) [reply packages/j/jos-animation.json:38]: Gotchya

**Review by MattIPv4 — APPROVED** [review]
LGTM

**Comment by jesvijonathan** [comment]
@MattIPv4, any idea why the latest of [JOS](https://www.npmjs.com/package/jos-animation) from npm is not appearing in [CDNJS](https://cdnjs.com/libraries/jos-animation) ?

also the version options are limited...

**Pull request merged**